docs/tutorials/near-indexer.md

```diff
@@ -60,32 +60,52 @@ echo 1.51.0 > rust-toolchain
 near-indexer = { git = "https://github.com/near/nearcore" }
 ```
 
-> **Note:** While it is fine to omit specific commit hash, for this tutorial we highly recommend to freeze near-indexer dependency for specific commit from the `nearcore` repository. _(Example below)_
+:::note
+While it is fine to omit a specific commit hash, for this tutorial we highly recommend freezing near-indexer dependency to a specific commit from the `nearcore` repository. _(Example below)_ It could also be useful to reference the [near-indexer-for-explorer repo](https://github.com/near/near-indexer-for-explorer/blob/master/Cargo.toml) for commit version and dependency versions they utilize.
```

Comment:
```suggestion
While it is fine to omit a specific commit hash, for this tutorial we highly recommend freezing near-indexer dependency to a specific commit from the `nearcore` repository. _(Example below)_

It could also be useful to reference the [near-indexer-for-explorer repository](https://github.com/near/near-indexer-for-explorer/blob/master/Cargo.toml) for commit version and dependency versions they utilize.
```